README.md

```diff
@@ -231,6 +231,30 @@ console.log('index name =', indexName)
 
 For more detailed information, see the [indexing strategies page](https://docs.mongodb.com/manual/applications/indexes/).
 
+## Error Handling
+
+If you need to filter certain errors from our driver we have a helpful tree of errors described in [docs/errors.md](https://github.com/mongodb/node-mongodb-native/blob/4.1/docs/errors.md).
+We expect `instanceof` checks to be maintained as our error API under semver as described in the hierarchy errors.md document.
+This means we can change the contents of error messages as we see fit to increase the helpfulness of a given message.
```

Comment:
I think this bit could be more clear, maybe leading with the first part. Something along the lines of:

"It is our recommendation to avoid relying on parsing error message strings in your code, preferring `instanceof` checks. We guarantee the class names of our errors won't change according to semver guidelines, but error messages are subject to change at any time, even patch releases, as we see fit to increase the helpfulness of a given message."

Reply:
What do you think of my suggestion above regarding subclass phrasing? (since technically we did introduce subclasses outside a major release as the errors work went into a minor version)